_layouts/default.html

```diff
@@ -34,9 +34,11 @@
             <br>
             <div class="menu">
         	<a href="/"><img src="/assets/images/home.png" height="15px"></a>
+                <a href="https://petstore.swagger.io/?url=https://raw.githubusercontent.com/Materials-Consortia/optimade-python-tools/master/openapi.json" target="_blank">API Documentation
```

Comment:
```suggestion
<a href="https://petstore.swagger.io/?url=https://raw.githubusercontent.com/Materials-Consortia/optimade-python-tools/master/openapi/openapi.json" target="_blank">API Documentation
```

Reply:
Looks like our last PR changed this link :)

Reply:
thanks for catching this!